app/frontend/src/components/FormList.vue

```diff
@@ -51,20 +51,23 @@ export default {
     // TODO: Consider loading part of this from API?
```

Comment:
There is an api for that, and forms are marked active (enabled), but we may want some more metadata like long title, short title. Not sure what you're doing with linkAdminName and linkName - whether that has any bearing on actual file/folder names.

Reply:
The linkname is the vue-router name alias for what view to navigate towards. Haven't quite figured out how we'd want to manage that if we are pulling information from the backend as well. There's definitely a naming scheme pattern, but I don't know if we should tightly couple and store that into the backend database and have it be pulled up from there just yet.